src/components/ChangePlan/Checkout/ContactInformation/ContactInformation.test.js

```diff
@@ -470,6 +520,114 @@ describe('Checkout component', () => {
 
       const inputPhone = container.querySelector('input#phone');
       fireEvent.change(inputPhone, { target: { value: '+54 223 655-8877' } });
+
+      const selectSecurityQuestion = container.querySelector('select#securityquestion');
+      fireEvent.change(selectSecurityQuestion, { target: { value: '1' } });
+
+      const inputAnswerSecurityQuestion = container.querySelector('input#answerSecurityQuestion');
+      fireEvent.change(inputAnswerSecurityQuestion, { target: { value: 'Answer' } });
+    });
+
+    await act(async () => {
+      const submitButton = container.querySelector('button[type="submit"]');
+      fireEvent.submit(submitButton);
+    });
+
+    // Assert
+    expect(getByText('validation_messages.error_required_field')).toBeInTheDocument();
+  });
+
+  it('should show error message if security question field is empty', async () => {
```

Comment:
This test is duplicated. It's not necessary because already exists.

Reply:
@jcamposmk Reviewing the existing code, I found that the industry selection had its own test and followed that line, but if you think it is redundant, I can remove it